sdk/keyvault/Azure.Security.KeyVault.Certificates/src/CertificateOperation.cs

```diff
@@ -84,7 +101,11 @@ public override KeyVaultCertificateWithPolicy Value
 
         /// <summary>
         /// Updates the status of the certificate operation.
+        /// This operation requires the certificates/get permission.
```

Comment:
This is not blocking the PR, but are we throwing an exception with a good error message when the caller does not have the right permissions? It would be great if the exception pointed people to some document describing how to acquire and manage permissions.

Reply:
No. Keys, Secrets, and Certificates just throw a `RequestFailedException` with HTTP 403 and an error message that is descriptive, though not great. Do other track 2 client libraries wrap and re-throw (or handle 403s specifically)?